Product/SoapUI_Test/RegressionSuite/Standard/AuditLogging-Standard/PatientDiscoveryDeferredResponseAuditLog.properties

```diff
@@ -51,7 +40,7 @@ ActiveParticipant_Dest.RoleIDCode.@code=110152
 ActiveParticipant_Dest.RoleIDCode.@codeSystemName=DCM
 ActiveParticipant_Dest.RoleIDCode.@displayName=Destination
 
-AuditSourceIdentification.@AuditEnterpriseSiteID=Gateway 1
+AuditSourceIdentification.@AuditEnterpriseSiteID=DoD
 AuditSourceIdentification.@AuditSourceID=urn:oid:1.1
 AuditSourceIdentification.@AuditSourceTypeCode=1010
 
```

Comment:
We decided to have setup script at g0 and g1 suite level for both Standard and Passthrough soapui tests. It's not seen at suite level setup and teardown scripts.